src/Pages/Dashboard/ShowHistoryGraph.jsx

```diff
@@ -0,0 +1,53 @@
+import React, {PureComponent} from "react";
+import {Paper, Typography, FormControl, InputLabel, Select, MenuItem, Grid, withStyles} from "@material-ui/core"
+import {historyStyle} from "./dashboardStyle";
+
+class ShowHistoryGraph extends PureComponent {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        const {classes} = this.props;
+
+        this.state = this.props.state;
```

Comment:
I don't think it's necessary to extract these variables out before using them in your render return, especially since you only use them in one place anyway.